### ISSUE TITLE
Ability to send folder

### ISSUE THREAD

**Issue by Tarik02** [telegramdesktop/tdesktop#5356]
Just drag and drop folder to window, like just file. Telegram Desktop should just put the folder into zip archive and send.

**Comment by john-preston** [comment]
@Tarik02 No, Telegram Desktop does not do that. Maybe you're talking about the macOS native app, it recently added such option.

**Comment by Tarik02** [comment]
Yes, I heard about it and I want to have this in Telegram Desktop. (This is Feature Request)

**Comment by omkarnathsingh** [comment]
Sounds a nice feature, can it be implemented in TD, that's the point.

**Comment by omkarnathsingh** [comment]
@Aokromes ? It got autoclosed. Should it not be in backlog?
